README.md

```diff
@@ -7,14 +7,13 @@ usage, and links to related Hoist and Toolbox source code. Toolbox also provides
 examples which may be especially useful as a starting point for application developers new to the
 framework.
 
-Please refer to the [Hoist Core](https://github.com/xh/hoist-core) and
-[Hoist React](https://github.com/xh/hoist-react) repos for detailed information and documentation on
-Hoist, including more information on configuring a local development environment.
+Please refer to the [Hoist Core](https://github.com/xh/hoist-core) and [Hoist React](https://github.com/xh/hoist-react) repos for detailed information and
+documentation on Hoist, including more information on configuring a local development environment.
 
 Toolbox is itself a Hoist Application, and we keep it updated with the latest versions of the Hoist
-Framework. A [toolbox-dev instance](https://toolbox-dev.xh.io) is auto-deployed via Teamcity on each
-commit to either the Toolbox or Hoist `develop` branches. We update a distinct
-["production" instance](https://toolbox.xh.io) manually with each new versioned Hoist release.
+Framework. A [toolbox-dev instance](https://toolbox-dev.xh.io) is auto-deployed via Teamcity on each commit to either the
```

Comment:
Not your change, but link `Teamcity` to a [remote source](https://www.jetbrains.com/teamcity/?source=google&medium=cpc&campaign=AMER_en_US-PST+MST_TeamCity_Branded&term=teamcity&content=513031283514&gad_source=1&gclid=Cj0KCQjws-S-BhD2ARIsALssG0YIFLHx7-SjM5spUH4tV0PH1TguOVkh9zVY0o7RhZmdDntgPqwiEv4aAuf6EALw_wcB? )? Maybe most people know what this is, but I was only previously aware of Jenkins, Bamboo, Travis, CircleCI and AWS CodePipeLine. When I first read this I was unsure what Teamcity meant here.